### DISCUSSION TITLE
single-cell-training/2022.7/day1/introduction_cellranger/

### DISCUSSION THREAD

**Discussion by giscus** [sib-swiss/single-cell-training#29] [Q&A]
# single-cell-training/2022.7/day1/introduction_cellranger/

https://sib-swiss.github.io/single-cell-training/2022.7/day1/introduction_cellranger/

**Comment by cperalta-viteri** [comment]
how can we get the data already preprocessed with cellranger? I would love to follow the course independently. Thank you

**Reply by GeertvanGeest** [reply]
Hi! How to install packages and download the data is described in the [Setup section](https://sib-swiss.github.io/single-cell-training/2022.7/day1/setup/). You will also find the [download link](https://single-cell-transcriptomics.s3.eu-central-1.amazonaws.com/course_data.tar.gz) there.

**Reply by cperalta-viteri** [reply]
Thank you so much!
